hw/ip/adc_ctrl/data/adc_ctrl.hjson

```diff
@@ -77,6 +77,10 @@
           desc: "Oneshot mode does not care about the filter value. 1'b0: disable; 1'b1: enable",
         }
       ]
+      tags: [
+        // Writes to ADC_EN_CTRL will cause side effects
+        "excl:CsrNonInitTests:CsrExclWrite:CsrExclWriteCheck"
+      ]
```

Comment:
I will suggest moving this to the `adc_enable` field instead.

Reply:
Moved the tags to adc_en_ctl.adc_enable V1 regression passes 100%

```
## ADC_CTRL Simulation Results
### Thursday January 20 2022 10:02:45 UTC
### GitHub Revision: [`e55a8c9c0`](https://github.com/lowrisc/opentitan/tree/e55a8c9c0ef63a9ceca99ec2333138d419cdd646)
### Branch: issue_10113
### [Testplan](https://docs.opentitan.org/hw/ip/adc_ctrl/doc/dv/#testplan)
### Simulator: XCELIUM

### Test Results
|  Milestone  |            Name            | Tests                               |  Passing  |  Total  |  Pass Rate  |
|:-----------:|:--------------------------:|:------------------------------------|:---------:|:-------:|:-----------:|
|     V1      |           smoke            | adc_ctrl_smoke                      |    50     |   50    |  100.00 %   |
|     V1      |        csr_hw_reset        | adc_ctrl_csr_hw_reset               |     5     |    5    |  100.00 %   |
|     V1      |           csr_rw           | adc_ctrl_csr_rw                     |    20     |   20    |  100.00 %   |
|     V1      |        csr_bit_bash        | adc_ctrl_csr_bit_bash               |     5     |    5    |  100.00 %   |
|     V1      |        csr_aliasing        | adc_ctrl_csr_aliasing               |     5     |    5    |  100.00 %   |
|     V1      | csr_mem_rw_with_rand_reset | adc_ctrl_csr_mem_rw_with_rand_reset |    20     |   20    |  100.00 %   |
|     V1      |                            | **TOTAL**                           |    105    |   105   |  100.00 %   |
|     V2      |  tl_d_outstanding_access   | adc_ctrl_csr_rw                     |    20     |   20    |  100.00 %   |
|             |                            | adc_ctrl_csr_aliasing               |     5     |    5    |  100.00 %   |
|             |                            | adc_ctrl_csr_hw_reset               |     5     |    5    |  100.00 %   |
|     V2      |    tl_d_partial_access     | adc_ctrl_csr_rw                     |    20     |   20    |  100.00 %   |
|             |                            | adc_ctrl_csr_aliasing               |     5     |    5    |  100.00 %   |
|             |                            | adc_ctrl_csr_hw_reset               |     5     |    5    |  100.00 %   |
|             |                            | **TOTAL**                           |    105    |   105   |  100.00 %   |
```